### PR TITLE
js-unit-testing_get-animal done

### PR DESCRIPTION
**Pull request by PARAMET7**

### CODE REVIEW

File: 09-01-24/I-like-animals/js-unit-testing_get-animal/index.js [hunk 1]
```diff
@@ -1 +1,9 @@
-export function getAnimal() {}
+export function getAnimal(animal) {
+  return animal === 'cats'
```
Comment (tonelloalan) [inline 09-01-24/I-like-animals/js-unit-testing_get-animal/index.js:2]: you should start with an `if` conditional. 
`if (animal === "cats")

File: 09-01-24/I-like-animals/js-unit-testing_get-animal/index.js [hunk 2]
```diff
@@ -1 +1,9 @@
...
+  ? 'I like animal'
```
Comment (tonelloalan) [inline 09-01-24/I-like-animals/js-unit-testing_get-animal/index.js:3]: ...and the return should show the requested phrase:
`return "I love cats!`

**Review by tonelloalan — CHANGES_REQUESTED** [review]
Try using if, if else, else conditionals to resolve it :)

File: 09-01-24/I-like-animals/js-unit-testing_get-animal/index.js (hunk 1 above)
Comment (PARAMET7) [reply 09-01-24/I-like-animals/js-unit-testing_get-animal/index.js:2]: it depending of the syntax, the "?" if..
 does it work to you?

**Pull request merged**